packages/core/src/components/table/table-body-row/table-body-row.tsx

```diff
@@ -70,6 +87,24 @@ export class TdsTableBodyRow {
     });
   }
 
+  handleRowClick(e) {
+    if (!this.clickable) return;
+    let rowIndex = Array.from(this.host.parentElement.children).indexOf(this.host);
+
+    this.tdsClick.emit({
+      event: e,
+      tableId: this.tableId,
+      rowIndex: rowIndex,
+    });
+  }
+
+  handleKeyDown(e) {
+    if (this.clickable && (e.key === 'Enter' || e.key === ' ')) {
+      e.preventDefault();
```

Comment:
Why is this needed?

Reply:
Good point @ckrook!

Reply:
I was more thinking of the `e.preventDefault()`, why is that part needed?

Reply:
The purpose of e.preventDefault() here is to prevent any default behavior associated with the Enter key or space, which could potentially trigger unintended actions in the browser (like submitting a form or scroll the page)

Reply:
Smart! I'd maybe just leave a comment stating this, just for the future. Feels like a thing that could easily be wrongfully removed in a refactor 🚀